euth/phases/models.py

```diff
@@ -1,15 +1,29 @@
 from django.conf import settings
 from django.db import models
-from model_utils import models as model_utils
 
 from euth.modules import models as modules_models
+from .validators import validate_content
+from . import content
+
+class PhasesManager(models.Manager):
```

Comment:
To be more flexible you should use a `QuerySet` instead of a `Manager`. `QuerySet` calls can be chained, what you can't do with a `Manager`.

This works only with a `QuerySet`:
```python
Phase.objects.all_phases().filter(description__icontains="foo").active_phase()
```
If you use a `Manager` the `QuerySet` created by `filter()` will not have the `active_phase()` method.

The custom `QuerySet` would look like this:

```python
class PhaseQuerySet(models.QuerySet):

    def active_phase(self, project):
        return self.filter(module__project=project).order_by('type').first()

    def all_phases(self, project):
        return self.filter(module__project=project).order_by('type')
```

And will be added to the model as a manager as follows:

```python
class Phase(models.Model):

    # fields

    objects = PhaseQuerySet.as_manager()
```

For more details see the [Creating a manager with QuerySet methods](https://docs.djangoproject.com/en/1.9/topics/db/managers/#creating-a-manager-with-queryset-methods).